HW0/P4/.ipynb_checkpoints/CS 205 HW0 P4-checkpoint.ipynb

```diff
@@ -0,0 +1,256 @@
+      "text/plain": [
+       "<matplotlib.figure.Figure at 0x10e283ed0>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "%matplotlib inline\n",
+    "import numpy as np\n",
+    "import matplotlib.pyplot as plt\n",
+    "from mpl_toolkits.mplot3d import Axes3D\n",
+    "\n",
+    "\n",
+    "if __name__ == '__main__':\n",
+    "    # Model parameters\n",
+    "    K = 121    # Number of times steps\n",
+    "    dt = 0.01  # Delta t\n",
+    "    c = 0.1    # Coefficient of drag\n",
+    "    g = -9.81  # Gravity\n",
+    "    # For constructing matrix B\n",
+    "    bx = 0\n",
+    "    by = 0\n",
+    "    bz = 0\n",
+    "    bvx = 0.25\n",
+    "    bvy = 0.25\n",
+    "    bvz = 0.1\n",
+    "    # For constructing matrix C\n",
+    "    rx = 1.0\n",
+    "    ry = 5.0\n",
+    "    rz = 5.0\n",
+    "    rvector=[rx, ry, rz]\n",
+    "\n",
+    "    # Create 3D axes for plotting\n",
+    "    ax = Axes3D(plt.figure())\n",
+    "    \n",
+    "#####################\n",
+    "    # Part 1:\n",
+    "    # Load true trajectory and plot it\n",
+    "data = np.loadtxt('P4_trajectory.txt',delimiter=',')\n",
+    "\n",
+    "print (len(data))\n",
+    "\n",
+    "xcoords=[]\n",
+    "ycoords=[]\n",
+    "zcoords=[]\n",
+    "vxcoords=[]\n",
+    "vycoords=[]\n",
+    "vzcoords=[]\n",
+    "\n",
+    "\n",
+    "for i in range(len(data)):\n",
```

Comment:
There are ways to separate out the x-coords, y-coords, and z-coords without looping. ex: xcoords = data[:,0]